scripts/populate-debug-workspaces.sh

```
@@ -0,0 +1,101 @@
+  if [ -z "$WS_ID" ]; then
+    echo "  [skip] Failed to create workspace '$name'"
+    continue
+  fi
+  sleep 0.3
+
+  # Rename
+  cmd rename-workspace "$WS_ID" "$name" || true
+  sleep 0.2
+
+  # Get the default surface
+  SURFACES=$(cmd list-panes --workspace "$WS_ID" 2>/dev/null || true)
```

Shellcheck (0.11.0)

[warning] SURFACES appears unused. Verify use (or export if used externally).

(SC2034)